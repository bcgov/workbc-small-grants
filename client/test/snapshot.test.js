import React from 'react';
import Link from '../Link.react';
import renderer from 'react-test-renderer';

test('renders correctly', () => {
  const component = renderer.create(
  <Link page="https://workbc-grants-dev.pathfinder.gov.bc.ca/">WorkBC Grants</Link>,
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();

});
