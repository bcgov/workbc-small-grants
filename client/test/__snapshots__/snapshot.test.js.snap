// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<a
  className="normal"
  href="https://workbc-grants-dev.pathfinder.gov.bc.ca/"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
>
  WorkBC Grants
</a>
`;
